components/automate-ui/src/app/helpers/infra-proxy/collapsible-list-mapper.ts

```diff
@@ -0,0 +1,15 @@
+import { Menu } from './../../entities/cookbooks/cookbook-details.model';
```

Comment:
Please use paths without `..`: 
```suggestion
import { Menu } from 'app/entities/cookbooks/cookbook-details.model';
```